api/app/Repositories/JWTRefreshToken/JWTRefreshTokenRepository.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Repositories\JWTRefreshToken;
+
+use App\Entity\JWTRefreshToken;
+use App\Repositories\BaseRepository;
+
+final class JWTRefreshTokenRepository extends BaseRepository implements JWTRefreshTokenRepositoryInterface
+{
+    public function getById(int $id): ?JWTRefreshToken
+    {
+        return JWTRefreshToken::find($id);
+    }
+
+    public function getByUserId(int $userId): ?JWTRefreshToken
+    {
+        return JWTRefreshToken::firstWhere('user_id', $userId);
+    }
```

Comment:
Users can have more than one refresh token, so there is no sense to get only first. I think we can remove this for now.

Reply:
Change firstWhere on Where or remove this method?

Reply:
@yevhen-turovets remove the method.